.github/CONTRIBUTING.md

```diff
@@ -6,19 +6,23 @@ Follow the [Meme Battle styleguide](https://github.com/Meme-Battle/styleguide/bl
 
 ```bash
       NEXT_STRUCTURE
-      ├── README.md - You are here!        
+      ├── README.md - You are here!   
+      ├── locales/ # Languages items
+      │      └── # Start file names with lowercase letters ex: en.json
       ├── src/
-      │   ├── app/
-      │   │   └── page-example/ # Use lowercase letters in page names
-      │   │   │      └── page.tsx # Lower case file name  ex: page.tsx
-      │   │   │      └── styles.module.scss # Style name must follow pattern styles.module.scss
-      │   │   └── page-example-two/ 
-      │   │          └── page.tsx
-      │   │          └── styles.module.scss
+      │   ├──app/
+      │   │   └──[lang]       
+      │   │   │        └── page-example/ # Use lowercase letters in page names
+      │   │   │        │        └── page.tsx # Lower case file name  ex: page.tsx
+      │   │   │        │        └── styles.module.scss # Style name must follow pattern styles.module.scss
+      │   │   │        └── page-example-two/ 
+      │   │   │                 └── page.tsx
+      │   │   │                 └── styles.module.scss
+      │   │   └── midleware.ts # Use lowercase letters in middleware 
```

Comment:
`middleware.ts` is not into `app` folder, it is into `src`, the project files is not following this pattern.